src/dynamic-pages/analyze/charts/chart.tsx

```diff
@@ -69,6 +69,20 @@ export function withChart<T = unknown, P = {}>(useOption: (props: DangerousCtx<T
     if (ctx.context.DEBUG_PRINT_OPTION) {
       console.debug(option)
     }
+    // show no data
+    if (isNoData(ctx as AnalyzeChartContextProps)) {
+      option.graphic = {
+        type: 'text',
+        left: 'center',
+        top: 'middle',
+        style: {
+          fontSize: 16,
+          fontWeight: 'bold',
+          text: "No event yet",
```

Comment:
how about use "No relevant data yet" @sykp241095

Reply:
No idea...

Reply:
Empty set

Reply:
@ChenlingLu `No data` is okay.